app/models/concerns/court/postcode_courts.rb

```diff
@@ -46,6 +47,18 @@ def add_pc_court(postcode_court, postcode)
             Please remove it from this court before assigning it to #{name}."
         end
       end
+
+      def validate_and_add_new_pc_court(postcode)
```

Comment:
Given the long name and the word 'and' in the middle - is this method doing too much ?